packages/data-models/src/language_model.js

```diff
@@ -378,7 +378,7 @@ class LanguageModel {
    * @returns {string} A normalized word.
    */
   static normalizeTrailingDigit (word) {
-    return /^.+1$/.test(word) ? word.substring(0, word.length - 1) : word
+    return /^.+\d$/.test(word) ? word.substring(0, word.length - 1) : word
```

Comment:
this is the fix for #599 (at least partially, there may be more to do for that issue). We should ignore all trailing digits when comparing words, not just a 1